grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -42,8 +42,9 @@ class ClientBatchJob {
     public static final String WORKFLOW_CONFIGS = "workflowConfigIds"
```

Comment:
Not really related to your change, but wondering in retrospect if a builder really makes sense if you only ever have one client, and are building in one fashion.

Seems every change to this file endures lots of development overhead to maintain pattern.